### ISSUE TITLE
Moonlight won't start up RGB Max 2

### ISSUE THREAD

**Issue by mjkbibi** [southoz/RetroOZ#49]
When starting up moonlight steam via rgb max 2, the screen turns black then fades back to the main menu. This is on 0.52b build.

**Comment by Deathtickles** [comment]
Seeing this same issue. Additionally, the generated .sh files are completely blank if you open them with an editor.

A freshly flashed SD does not have this issue so that was my only work-around.

0.52b

**Comment by mjkbibi** [comment]
Seeing that too. If you're referring to moonlight streaming working on a freshly flashed SD, I wonder if you do it a specific way or format? I only installed RetroOZ several times on a new SD card and formatted it and tried again to no avail.

**Comment by southoz** [comment]
Sorry, don't have a Max 2 but this issue is general as the files are all blank.

In general, the files are specific to my testing, users should generate their own files with the generate files function after connecting to a server.

Typical workflow is Pair -> List - Generate

I test with Geforce Experience so I can not validate the Radeon / Sunshine solution.

I will release a new update when I get a chance to update the files and integrate the Max 2 L1/L2 R1/R2 dtb update.

**Comment by southoz** [comment]
moonlight-steam.sh

```
#!/bin/bash
perfmax none
moonlight stream -app Steam -config /roms/moonlight/moonlight.conf -mapping /roms/moonlight/gamecontrollerdb.txt
perfnorm none
```

**Comment by nfriedly** [comment]
I just tried out moonlight for the first time today and ran into the same issue - nothing will launch. 

I tried ssh'ing into my device (pro tip: default username/password is `odroid`/`odroid` - that should really be documented somewhere) and I noticed that there were a few files in my home folder that were basically all but the first word of each game moonlight was aware of. So, for example, "Fallout 4 Far Harbor" led to empty files named `4`, `Far`, and `harbor.sh` in `/home/odroid`. I'm guessing the filename should have been in quotes somewhere and wasn't.

That may not be the only issue, but it's definitely an issue.

Update: `/roms/moonlight/Fallout` is an empty file also :/

**Comment by nfriedly** [comment]
Oh, I figured out one more issue: It was missing the `gamecontrollerdb.txt`. Fortunately there was a backup that I was able to restore like so:

`cp /usr/local/share/moonlight/gamecontrollerdb.txt /roms/moonlight`

After that, I was able to launch Dead Cells, although I had to use the mouse at the computer to bring the game to the foreground.

Additionally, once I exited the game, moonlight dropped me onto my desktop, but without any mouse input. So there was no obvious way to launch anything new or close moonlight. 

(Although, pressing the-button-that's-labeled-Select-but-isn't a couple of times brought up Steam Big Picture mode, which does support controller navigation, so I suppose I could launch a new steam game that way.)

I could probably also install controller companion or something similar on the desktop, but is there a way to exit moonlight from the device?
